functions/package.json

```diff
@@ -25,16 +25,16 @@
   "dependencies": {
     "escape-string-regexp": "^4.0.0",
     "firebase-admin": "^9.12.0",
-    "firebase-functions": "^3.16.0"
+    "firebase-functions": "3.16.0"
```

Comment:
Got build errors that I didn't try to figure out when I tried updating further.

Reply:
What do you think about adding this comment to the `dependencies-notes.md`? 

That currently seems to just be about the SPA dependencies, so maybe it would be bad to add it there unless you also reviewed all of the other dependencies for the functions too.  I don't have a preference, I just remembered that file I saw this comment.

Reply:
> What do you think about adding this comment to the `dependencies-notes.md`?

Fixed -- apparently 3.17.0 was just a broken release, as indicated by the description of the [3.17.1](https://github.com/firebase/firebase-functions/releases/tag/v3.17.1) release.